ConsoleCalculator.Tests/CalculatorFixture.cs

```diff
@@ -1,14 +1,178 @@
-using System;
 using Xunit;
 
 namespace ConsoleCalculator.Tests
 {
     public class CalculatorFixture
     {
+        Calculator calculator = new Calculator();
+
+        private string SendMultipleKey(char[] key)
+        {
+            var i = 0;
+            for (i = 0; i < key.Length - 1; i++)
+                calculator.SendKeyPress(key[i]);
+            return calculator.SendKeyPress(key[i]);
+        }
+
+
         [Fact]
-        public void DummyTest()
+        public void TestKeyPress()
         {
-            return;
+            char key = '1';
+
+            string fromCall = calculator.SendKeyPress(key);
+
+            Assert.Equal("1", fromCall);
+        }
+
+        [Fact]
+        public void TestMultipleKeyPress()

```

Comment:
For thid method "MultipleKeyPress_With_Different_Values_Should_Return_12". update all the method names.